pyaedt/modeler/GeometryOperators.py

```diff
@@ -1448,3 +1459,468 @@ def orient_polygon(x, y, clockwise=True):
             x.reverse()
             y.reverse()
         return x, y
+
+    @staticmethod
+    @aedt_exception_handler
+    def v_angle_sign(va, vb, vn, righthanded=True):
```

Comment:
Just to be more pythonic
```suggestion
    def v_angle_sign(va, vb, vn, right_handed=True):
```